doc/todo/RELEASE.md

```diff
@@ -36,6 +36,7 @@ If no release critical problems are found during testing, continue.
 ### Check
 
 - https://build.libelektra.org/job/libelektra/job/master/ must pass
+- Run `scripts/dev/audit-depndencies` to make sure no known vulnerabilities exist in project dependencies
```

Comment:
There is a typo. And it would be better if its a link, also for both the script+md file in the release notes.

```suggestion
- Run [audit-dependencies](/scripts/dev/audit-dependencies) to make sure no known vulnerabilities exist in dependencies
```